documentation/status-logging.md

```diff
@@ -0,0 +1,54 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging though you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
+
+    ```
+    status: {
+        logging-frequency: 2000,
+        status-logging-enabled: true
+    }
```

Comment:
I guess I have a slight preference for ditching the `enabled` setting, keeping the section called `status` (or, alternately, `status-service`, and I don't know how to decide between those :( ).  And maybe I'd call the interval setting `debug-logging-interval`.

Man I hate naming things.  @rlinehan do you have any opinions on this?  You're always good at catching the most idiotic things about my naming suggestions :)